modules/llama_cpp_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13)
+
+project(LlamaCppPlugin)
+
+find_package(OpenVINODeveloperPackage REQUIRED)
+
+ov_option(ENABLE_LLAMA_CPP_PLUGIN_REGISTRATION "Enables registration of LLAMA_CPP plugin" ON)
```

Comment:
let's currently disable registration, because it will affect main OV build results